src/gui/core/plugin/userui/pom.xml

```diff
@@ -132,7 +132,15 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+        
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
+			<version>1.7.7</version>
+			<scope>test</scope>
+		</dependency>
 
```

Comment:
no, it was just for the not meanigfull test discussed in this pul req, I'll remove it